### DISCUSSION TITLE
Enable modular imports instead of just `import {} from '@ark-ui/react'`

### DISCUSSION THREAD

**Discussion by ivancuric** [chakra-ui/ark#2817] [General]
Currently everything from arkui is loaded during development in vite, making inspection really difficult.

**Comment by segunadebayo** [comment]
Modular imports are already supported in Ark

import { Accordion } from ark-ui/react/accordion

**Comment by ivancuric** [comment]
Does this work with solid? Haven't managed to make the imports work 🤔

**Reply by segunadebayo** [reply]
Yeah, it doesn't work in Solid.js just yet. We'll look into that.

**Comment by segunadebayo** [comment]
Closing — modular imports work across all frameworks:

```tsx
// React
import { Accordion } from '@ark-ui/react/accordion'

// Solid
import { Accordion } from '@ark-ui/solid/accordion'

// Vue
import { Accordion } from '@ark-ui/vue/accordion'
```
